runtime/include/scheduler.h

```diff
@@ -122,6 +162,8 @@ static inline struct sandbox *
 scheduler_get_next()
 {
 	switch (scheduler) {
+	case SCHEDULER_MTDS:
```

Comment:
If this list continues to grow, doing dispatch with switches won't scale well at all. Maybe a concern for the future, but it would be good to understand how long it takes to schedule to make a more fact-based decision.